packages/components/organisms/f-checkout/CHANGELOG.md

```diff
@@ -3,6 +3,16 @@
 The format is based on [Keep a Changelog](http://keepachangelog.com/en/1.0.0/)
 and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.html).
 
+
+
+v0.68.1
+------------------------------
+*March 9, 2021*
+
+### Changed
+- Locale usage in storybook
+
+
```

Comment:
Is it only f-checkout and f-searchbox that are using Locale in storybook or are there any other components that might need updating?

Reply:
Fortunately these are the only usages.